### PR TITLE
feat: styling for call feedback

### PR DESCRIPTION
**Pull request by shabanov-dlg**

### CODE REVIEW

File: src/components/CallFeedback/CallFeedbackRating.js [hunk 1]
```diff
@@ -6,25 +6,34 @@
...
 import CallFeedbackRatingStar from './CallFeedbackRatingStar';
 import styles from './CallFeedback.css';
+import { Text } from '@dlghq/react-l10n';
 
 type Props = {
   id: string,
   value: number,
   maxRating: number,
   onChange: (value: number) => mixed,
+  labels?: string[],
```
Comment (meafmira) [inline src/components/CallFeedback/CallFeedbackRating.js]: You have default prop, remove optional

File: src/components/CallFeedback/CallFeedbackRating.js [hunk 2]
```diff
@@ -6,25 +6,34 @@
 import React, { PureComponent } from 'react';
 import CallFeedbackRatingStar from './CallFeedbackRatingStar';
 import styles from './CallFeedback.css';
+import { Text } from '@dlghq/react-l10n';
```
Comment (meafmira) [inline src/components/CallFeedback/CallFeedbackRating.js]: Move above to absolute imports

File: src/components/CallFeedback/CallFeedbackRatingStar.js [hunk 3]
```diff
@@ -7,32 +7,49 @@ import React, { PureComponent } from 'react';
 import classNames from 'classnames';
 import Icon from '../Icon/Icon';
 import styles from './CallFeedback.css';
+import { Text } from '@dlghq/react-l10n';
```
Comment (meafmira) [inline src/components/CallFeedback/CallFeedbackRatingStar.js]: Move to absolute imports

File: src/components/CallFeedback/CallFeedbackRatingStar.js [hunk 4]
```diff
@@ -7,32 +7,49 @@ import React, { PureComponent } from 'react';
...
 
 type Props = {
   value: number,
   active: boolean,
   onChange: (value: number) => mixed,
+  icon?: string,
```
Comment (meafmira) [inline src/components/CallFeedback/CallFeedbackRatingStar.js]: Remove all these optionals. You have default props

**Comment by meafmira** [comment]
@shabanov-dlg fix flow errors

File: src/components/CallFeedback/CallFeedbackRating.js (hunk 1 above)
Comment (shabanov-dlg) [reply src/components/CallFeedback/CallFeedbackRating.js]: Yes, my mistake.

**Pull request merged**